03/Product.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+function Product(props) {
+    const {content, item, onClick, disabled} = props;
+
+    return (
+        <li>
+            <p>{item.name}</p>
+            <p>{item.price}</p>
+            <button
+                content={content}
+                data={item}
+                onClick={onClick}
```

Comment:
Ponieważ w `app.js` przekazane funkcje mają parametry to trzeba je przekazać czyli powinno być: ` onClick={() => onClick(product)}`